service/controller.go

```diff
@@ -0,0 +1,16 @@
+package service
+
+import (
+	"context"
+
+	// pb "github.com/meateam/fav-service/proto"
```

Comment:
Why not use the structures we defined in the proto file?

Reply:
Done